src/main/java/life/qbic/projectwizard/control/WizardController.java

```diff
@@ -1198,16 +1198,19 @@ public void activeStepChanged(WizardStepActivationEvent event) {
           }
           Project p = openbis.getProjectByIdentifier("/" + space + "/" + proj);
           Map<String, List<Sample>> samplesByExperiment = new HashMap<String, List<Sample>>();
-          for (Sample s : openbis.getSamplesOfProject(p.getIdentifier())) {
-            String expID = s.getExperimentIdentifierOrNull();
-            String exp = expID.substring(expID.lastIndexOf("/") + 1);
-            if (samplesByExperiment.containsKey(exp)) {
-              List<Sample> lis = samplesByExperiment.get(exp);
-              lis.add(s);
-              samplesByExperiment.put(exp, lis);
+          for (Sample s : openbis.getSamplesOfProject(p.getIdentifier().getIdentifier())) {
```

Comment:
`s`, `p`
Also, I do wonder whether you want to do `getIdentifier().getIdentifier()` here or if it is in fact a copy-paste error.